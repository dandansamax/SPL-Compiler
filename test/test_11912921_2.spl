struct Pokemon
{
    char name[20];
    int hp;
};

struct Pokemon get_pokemon(int id)
{
    struct Pokemon pikachu;
    pikachu.hp = 50;
    return pikachu;
}

int test_2()
{
    struct Pokemon who = get_pokemon(1);
    who = 1;
    return 0;
}
