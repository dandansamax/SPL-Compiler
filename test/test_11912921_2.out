Error type A at Line 2: unknown lexeme @
Error type B at Line 2: Invalid array declaration
